Resources/template.json

```diff
@@ -154,24 +154,30 @@
         "description": "Custom host name for the website"
       }
     },
-    "certificateResourceGroup": {
+    "keyVaultName": {
       "type": "string",
       "defaultValue": "",
       "metadata": {
-        "description": "Resource group for the certificate - only used if appHostName is specified"
+        "description": "Key vault with the certificate in - only used if appHostName is specified"
+      }
+    },
+    "keyVaultResourceGroup": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().name]",
+      "metadata": {
+        "description": "Resource group for the key vault - only used if appHostName is specified"
       }
     },
     "certificateName": {
       "type": "string",
       "defaultValue": "",
       "metadata": {
-        "description": "Certificate for the custom host name - only used if appHostName is specified"
+          "description": "Array of certificates - only used if appHostName is specified"
```

Comment:
Should this have been changed?  Looks like it's still the name of the certificate for the custom hostname